01.prepare_dataset.py

```
@@ -9,31 +12,12 @@
-# # Evaluate the model
-# mse, r2 = model.evaluate(X_test, y_test)
-
-# ## Visualizing Results
-# y_pred = model.predict(X_test)
-# visualize_results(y_test, y_pred)
-
-# ## Feature Importance
-# feature_importance, feature_names = model.get_feature_importance()
-# plot_feature_importance(feature_importance, feature_names)
\ No newline at end of file
+data_processor.save_to_catalog(train_set=train_set, test_set=test_set, spark=spark)
```

Ruff

Undefined name `spark`

(F821)